### PR TITLE
progress on presentation for NW/post-fire research group

### PR DESCRIPTION
**Pull request by steeleb**
Hiya Katie!

This *looks* really big, but it's mostly data exploration code other than the presentation file. There's quite a bit of renaming/housekeeping (editing of READMEs) in the top half of the file changes.

There's a bunch of files that I made for prelim analysis (all in the subdirectory `programs/integrated-data`) that are drafts/exploration. There's already something I see in there that I need to fix (L83 tot he end of the`programs/integrated-data/0_secchi_LSC2SR_matchup.Rmd` script - this mapping is problematic and creates dupes - we've gone to 'overmatching' in other workflows that I need to implement here), but that will be in a future PR. Probably just a skim here, a lot of this is in flux, review for context?

The script `programs/integrated-data/1_secchi_LSC2SR_explore.Rmd` has a bunch of preliminary xgboost development, again, probably doesn't need thorough review, but maybe skim so you know what's up?

**Okay, the money review**: `programs/presentations/2023-04_RS_WQ_NW_prelim.qmd`:

This was a presentation we were supposed to give today, but then we pivoted because there weren't the necessary people in the room. I'll be using this as a template for my presentation on Tuesday which will focus on temperature and chl-a (and NOT Secchi, I'll have to fill you in on that in person at some point).

Let me know if you have questions/concerns?

### CODE REVIEW

File: programs/presentations/2023-04_RS_WQ_NW_prelim.qmd [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+
+## Motivation
+
+To build a water quality prediction system for the 12 Northern Water reservoirs using machine learning, remote sensing, and live-streaming sensors.
+
+## How do we accomplish this?
+
+We will use robust, peer-reviewed scientific research methods to create a decision support system:
+
+**Remote Sensing**:
```
Comment (kathryn-willi) [inline programs/presentations/2023-04_RS_WQ_NW_prelim.qmd:43]: **Remote Sensing:**

File: programs/presentations/2023-04_RS_WQ_NW_prelim.qmd [hunk 2]
```diff
@@ -0,0 +1,404 @@
...
+  theme_bw() +
+  scale_x_date(minor_breaks = '1 years')
+```
+
+</div>
+
+::: notes
+<div>
+
+We can now supplement the manual record using the `r nrow(rs_loc)` data points from the 7 lakes in our location-specific matchup dataset. The historical record goes from this
```
Comment (kathryn-willi) [inline programs/presentations/2023-04_RS_WQ_NW_prelim.qmd:265]: This sentence seems to drop off...?

**Review by kathryn-willi — APPROVED** [review]
When rendering Quarto docs, be sure to add:

`format:
  html:
    self-contained: true
    self-contained-math: true`

This allows htmls, etc. to render properly when you share/send them out to people! (For example, when I downloaded the current version of this, it did not include any of the maps).

File: programs/presentations/2023-04_RS_WQ_NW_prelim.qmd (hunk 2 above)
Comment (steeleb) [reply programs/presentations/2023-04_RS_WQ_NW_prelim.qmd:265]: the punch line is on the next slide! :)

**Comment by steeleb** [comment]
> When rendering Quarto docs, be sure to add:
> 
> `format: html: self-contained: true self-contained-math: true`
> 
> This allows htmls, etc. to render properly when you share/send them out to people! (For example, when I downloaded the current version of this, it did not include any of the maps).

That makes sense! Quarto also outputs a `..._files` folder that would make the .html work (I untracked that in this repo) this also provides the info for the animations in revealjs. I'll have to see if the animations work in the html with this addition because this is obviously a million times better.

**Pull request merged**